api/src/main/scala/za/co/absa/loginsvc/rest/config/actuator/GitConfig.scala

```diff
@@ -30,26 +30,30 @@ object GitPropertiesGenerator {
   private var branch: String = _
   private var commitId: String = _
   private var commitTime: String = _
+  private var latestVersion: String = _
   def generateGitProperties(writeFile: Boolean): Unit = {
     val dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
     setProperties(
       getGitOutput("git rev-parse --abbrev-ref HEAD").getOrElse("unknown"),
       getGitOutput("git rev-parse HEAD").getOrElse("unknown"),
-      dateFormat.format(getGitOutput("git show -s --format=%ct HEAD").map(_.toLong * 1000).getOrElse(0L))
+      dateFormat.format(getGitOutput("git show -s --format=%ct HEAD").map(_.toLong * 1000).getOrElse(0L)),
+      getGitOutput("git describe --tags --abbrev=0").getOrElse("unknown")
```

Comment:
```scala
setProperties(
      branch = getGitOutput("git rev-parse --abbrev-ref HEAD").getOrElse("unknown"),
      commitId = getGitOutput("git rev-parse HEAD").getOrElse("unknown"),
      commitTime = dateFormat.format(getGitOutput("git show -s --format=%ct HEAD").map(_.toLong * 1000).getOrElse(0L)),
      latestVersion = getGitOutput("git describe --tags --abbrev=0").getOrElse("unknown")
```

How about adding the param names here. They are all strings and can be easily mismatched, this also increased the readability.